src/topology/bornology/basic.lean

```diff
@@ -90,11 +90,15 @@ lemma is_cobounded_def {s : set α} : is_cobounded s ↔ s ∈ cobounded α := i
 
 lemma is_bounded_def {s : set α} : is_bounded s ↔ sᶜ ∈ cobounded α := iff.rfl
 
-lemma is_bounded_compl_iff {s : set α} : is_bounded sᶜ ↔ is_cobounded s :=
+@[simp] lemma is_bounded_compl_iff {s : set α} : is_bounded sᶜ ↔ is_cobounded s :=
 by rw [is_bounded_def, is_cobounded_def, compl_compl]
 
-@[simp]
-lemma is_bounded_empty : is_bounded (∅ : set α) :=
+@[simp] lemma is_cobounded_compl_iff {s : set α} : is_cobounded sᶜ ↔ is_bounded s := iff.rfl
```

Comment:
Doesn't this create simp-normal form issues?

Reply:
How so? The RHS is simpler than the RHS. There's no `ᶜ` on the RHS. You might be confused because `sᶜᶜ = s`, but that's no problem because they are not synctatically equal (not even defeq, actually).

Reply:
Yes, I thought there might be a problem with `sᶜᶜ = s`.